lightning/src/ln/channelmanager.rs

```diff
@@ -4494,68 +4494,56 @@ where
 		})
 	}
 
-	fn construct_pending_htlc_status<'a>(
-		&self, msg: &msgs::UpdateAddHTLC, counterparty_node_id: &PublicKey, shared_secret: [u8; 32],
-		decoded_hop: onion_utils::Hop, allow_underpay: bool,
-		next_packet_pubkey_opt: Option<Result<PublicKey, secp256k1::Error>>,
-	) -> PendingHTLCStatus {
-		macro_rules! return_err {
-			($msg: expr, $reason: expr, $data: expr) => {
-				{
-					let logger = WithContext::from(&self.logger, Some(*counterparty_node_id), Some(msg.channel_id), Some(msg.payment_hash));
-					log_info!(logger, "Failed to accept/forward incoming HTLC: {}", $msg);
-					if msg.blinding_point.is_some() {
-						return PendingHTLCStatus::Fail(HTLCFailureMsg::Malformed(
-							msgs::UpdateFailMalformedHTLC {
-								channel_id: msg.channel_id,
-								htlc_id: msg.htlc_id,
-								sha256_of_onion: [0; 32],
-								failure_code: LocalHTLCFailureReason::InvalidOnionBlinding.failure_code(),
-							}
-						))
-					}
-					let failure = HTLCFailReason::reason($reason, $data.to_vec())
-						.get_encrypted_failure_packet(&shared_secret, &None);
-					return PendingHTLCStatus::Fail(HTLCFailureMsg::Relay(msgs::UpdateFailHTLC {
-						channel_id: msg.channel_id,
-						htlc_id: msg.htlc_id,
-						reason: failure.data,
-						attribution_data: failure.attribution_data,
-					}));
+	fn construct_pending_htlc_fail_msg<'a>(&self, msg: &msgs::UpdateAddHTLC,
+		counterparty_node_id: &PublicKey, shared_secret: [u8; 32], inbound_err: InboundHTLCErr) -> HTLCFailureMsg {
```

Comment:
nit: we're trying to adopt rustfmt formatting where possible ahead of actually using it on this file, which would format as such:
```rust
fn construct_pending_htlc_fail_msg<'a>(
    &self, msg: &msgs::UpdateAddHTLC, counterparty_node_id: &PublicKey,
    shared_secret: [u8; 32], inbound_err: InboundHTLCErr,
) -> HTLCFailureMsg {
    ...
}